src/autocomplete/src/Autocomplete.js

```diff
@@ -199,8 +174,13 @@ export default class Autocomplete extends PureComponent {
 
     if (items.length === 0) return null
 
+    // Pass the actual DOM ref to downshift, this fixes touch support
+    const menuProps = getMenuProps()
```

Comment:
looks like some of these changes are in #793 too?

Reply:
Yep I realized we can ship these sooner, so I will remove them from this one.

Reply:
👍